jobs/JOBSPROC_MODS

```diff
@@ -126,30 +134,31 @@ export SENDECF=${SENDECF:-YES}
 # Paths for obsproc items
 # -------------------------------
 export HOMEobsproc=${HOMEobsproc:\
--${NWROOT:?}/obsproc.${obsproc_ver:?\
+-${PACKAGEROOT:?}/obsproc/${obsproc_ver:?\
 "###FATAL ERROR \"obsproc_ver\" is not set"}}
 
 # ---------------------------------------
 # Paths for dump and bufr_dumplist items
 # ---------------------------------------
-export HOMEbufr_util=${HOMEbufr_util:\
--${NWROOT:?}/bufr_util.${bufr_util_ver:?\
-"###FATAL ERROR \"bufr_util_ver\" is not set"}}
+export HOMEbufr_dump=${HOMEbufr_dump:\
+-${PACKAGEROOT:?}/bufr_dump.${bufr_dump_ver:?\
+"###FATAL ERROR \"bufr_dump_ver\" is not set"}}
```

Comment:
These 3 lines are redundant and not required; $HOMEbufr_dump is already pre-defined in the bufr_dump/1.0.0 module. I suggest removing them.